AsaApi/Core/Public/Ark/ArkApiUtils.h

```diff
@@ -783,14 +915,24 @@ namespace AsaApi
 		* \param _this Player controller
 		* \param Command Command to run
 		*/
-		void RunHiddenCommand(AShooterPlayerController* _this, FString* Command)
+		void RunHiddenCommand(AShooterPlayerController* _this, const FString* Command) const
```

Comment:
Before:
```
?RunHiddenCommand@IApiUtils@AsaApi@@QEAAXPEAUAShooterPlayerController@@PEAVFString@@@Z
```
After:
```
?RunHiddenCommand@IApiUtils@AsaApi@@QEBAXPEAUAShooterPlayerController@@PEBVFString@@@Z
```

Reply:
This is a purposeful breaking change to better reflect the intent of the function.

Recompiling plugins against the new api version would resolve the issue -- we can also force export this with the old name with a linker comment.

Reply:
Doesn't bother me :) I don't know if many plugins are even using `RunHiddenCommand()`. And I imagine `FVectorToCoords()` is mostly getting inlined anyways.

Personally, there's a lot of exports I wouldn't mind seeing broken (a whole lot of FString exports that could have just been defined in headers with `NativeCall`, among other things). Maybe if the dev team doesn't mind potentially breaking existing plugins, then there's hope for cleaning up some other exports as well.

Reply:
I agree, I am also not a fan of a singleton for ApiUtils.

This is just the first of a larger set of changes -- realistically we wouldn't push a new API version officially until most things are cleaned up and developers have time to recompile plugins.

With that said, we should also be using Git tags to help version the API so that people can download the latest stable source more easily, even if there are changes to master.